relay-event-schema/src/protocol/span.rs

```diff
@@ -299,6 +299,18 @@ pub struct SpanData {
     #[metastructure(field = "sentry.sdk.name")]
     pub sdk_name: Annotated<String>,
 
+    /// Slow Frames
+    #[metastructure(field = "frames.slow")]
+    pub frames_slow: Annotated<Value>,
```

Comment:
Do SDKs already send this field? We're trying to [align](https://github.com/getsentry/relay/pull/3457) field names in `span.data` with attribute names in OTel spans, so we should think about what the canonical name of this field should be. That is, If there's an [OTel convention](https://opentelemetry.io/docs/specs/semconv/) for it, use it, otherwise prefix with `sentry.`.

Reply:
SDKs [already send this field](https://develop.sentry.dev/sdk/performance/span-data-conventions/#mobile) and I don't believe there's a corresponding OTel convention so I will prefix it with `sentry.`